src/components/actions/ActionBar.tsx

```diff
@@ -27,13 +27,16 @@ export interface IActionBarOwnProps extends React.ClassAttributes<ActionBar> {
     removeDefaultContainerClasses?: boolean;
     withSmallActions?: boolean;
     prefixContent?: IContentProps;
+    width?: number;

```

Comment:
Isn't `width` supposed to be a string in order to be able to specify the css units (px, %, vw, ...) ?

Reply:
Here, it'll always be `px` since it's the width calculated. I wish I didn't have to use this, but the width is not predictable at all.